src/app/frontend/resource/cluster/node/list/template.html

```diff
@@ -0,0 +1,20 @@
+<!--
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<div>
+  <kd-graphs></kd-graphs>
```

Comment:
It is used only here. It should be used everywhere so we don't duplicate the code.

Reply:
I'd also rename it to something like 'kd-resource-status'

Reply:
@maciaszczykm  what do you mean!

Reply:
You have added component and used it only once. It should be used also from other views, not only from node list.

Reply:
yes, it's used in all views, but actually not  in overview and workloads and cluster because they already extends GroupedResourceList, and it's a duplication !

Reply:
Duplication: because the code of graphs is already used in GroupedResourceList, so do you want to use the new component instead!